corehq/messaging/scheduling/tests/test_recipients.py

```diff
@@ -655,6 +655,37 @@ def test_username_case_property_recipient(self):
             )
             self.assertIsNone(instance.recipient)
 
+    def test_user_id_case_property_recipient(self):
+        # test valid ID
+        with create_case(
+                self.domain,
+                'person',
+                owner_id=self.city_location.location_id,
+                update={'hq_user_id': self.web_user.get_id}
+        ) as case:
+            instance = CaseTimedScheduleInstance(
+                domain=self.domain,
+                case_id=case.case_id,
+                recipient_type=CaseScheduleInstanceMixin.RECIPIENT_TYPE_CASE_PROPERTY_USER,
```

Comment:
How difficult is it to add a new recipient type?  Like `RECIPIENT_TYPE_CASE_PROPERTY_USER_ID`.  It'd be nice to avoid the double lookup and force the configurer to specify